lab-21-JR/karma.conf.js

```diff
@@ -0,0 +1,72 @@
+// Karma configuration
```

Comment:
If you remove all of these comments, it'll make this config quite a bit easier to read.